src/System.Text.RegularExpressions/src/System/Collections/HashtableExtensions.cs

```diff
@@ -2,18 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics.CodeAnalysis;
+
 namespace System.Collections
 {
     internal static class HashtableExtensions
     {
-        public static bool TryGetValue<T>(this Hashtable table, object key, out T value)
+        public static bool TryGetValue<T>(this Hashtable table, object key, [MaybeNull] out T value)
```

Comment:
I think this should be `MaybeNullWhen(false)` instead.

Reply:
I believe existing values can also be null, based on how `HashTable` is currently annotated.